src/components/views/elements/Field.js

```diff
@@ -38,13 +41,13 @@ export default class Field extends React.PureComponent {
         delete extraProps.type;
         delete extraProps.placeholder;
         delete extraProps.label;
+        delete extraProps.element;
+
+        const element = this.props.element || "input";
+        const fieldInput = React.createElement(element, extraProps, this.props.children);
```

Comment:
`id`, `type`, and `placeholder` now need to pass through to the inner element via `extraProps`, so we shouldn't delete them anymore.

Also, maybe we should delete `children`, since it's handled separately.